sympy/geometry/ellipse.py

```diff
@@ -1022,6 +1022,33 @@ def auxiliary_circle(self):
         """
         return Circle(self.center, Max(self.hradius, self.vradius))
 
+    def director_circle(self):
+        """Returns a Circle consisting of all points where two perpendicular
+        tangent lines to the ellipse cross each other.
+
+        Returns
+        =======
+
+        Circle: A director circle returned as a geometric object.
```

Comment:
<img src="https://user-images.githubusercontent.com/34944973/54198062-e1b06700-4508-11e9-9cbf-ab8c047b873a.png" width="512px">

Actually, I find this part to be very sensitive to the location of colon, or such.
I think it should strictly be
```
Circle
    A director circle returned as a geometric object.
```

Reply:
yup that would be nice thanks @sylee957